clientd3d/annotate.c

```diff
@@ -175,6 +175,10 @@ void MapAnnotationClick(int x, int y)
       }
    }
 
+   // Clear any open tooltips
```

Comment:
Is this what TooltipReset() should do?  It's still called from DisplayDesrciption.  Do description dialogs have the same tooltip bug?

Reply:
I _believe_ `TooltipReset` was doing this at some point, maybe in older versions of Windows? And yes, this bug appears when description dialogs are opened while hovering over a map annotation.

I wonder: would it be overkill to update `TooltipReset` to simply do this:
```
// Clear any open tooltips
SendMessage(hToolTips, TTM_POP, 0, 0);
```
...and not bother with the mouse events?

Doing that and moving the `TooltipReset` calls up before the `DialogBoxParam` calls in these two places seems to fix the issue.